scripts/version_maker.sh

```diff
@@ -1,7 +1,11 @@
 #! /bin/bash 
 
-# parsing yaml file
-function yaml() {
+# Retrive the last pushed commit from the repo
+second_last_commit_hash=`git log -n 2 --pretty=format:"%H" | tail -1`
```

Comment:
What's the use case for fetch the 2nd last commit

Reply:
this is a guard against faulty pushes into the base template yaml

Reply:
okay